proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -75,6 +75,62 @@ message Dispatch {
 
   // The dispatch payload
   google.protobuf.Struct payload = 11;
+
+  // The recurrence rule
+  RecurrenceRule recurrence_rule = 12;
+}
+
+// Information about the schedule for reoccurring dispatches
+//
+// Example:
```

Comment:
To me sometimes the recurrencerule reads like an interval in which it should reoccur. Sometimes it reads like specific times at which it should reoccur. 
Maybe it makes sense to have a start time in combination with an interval?

Reply:
@TalweSingh this is derived from the iCal Standard.